lib/src/Uncertainty/Model/EllipticalDistribution.cxx

```diff
@@ -372,6 +361,9 @@ LevelSet EllipticalDistribution::computeMinimumVolumeLevelSetWithThreshold(const
   NumericalMathFunction minimumVolumeLevelSetFunction(MinimumVolumeLevelSetEvaluation(clone()).clone());
   minimumVolumeLevelSetFunction.setGradient(MinimumVolumeLevelSetGradient(clone()).clone());
   return LevelSet(minimumVolumeLevelSetFunction, -standard.computeLogPDF(point));
+  const NumericalScalar logThreshold = standard.computeLogPDF(point);
+  threshold = std::exp(logThreshold);
+  return LevelSet(minimumVolumeLevelSetFunction, -logThreshold);
```

Comment:
You probably wanted to remove the first one so as to assign the value of "threshold".